### ISSUE TITLE
为什么loss会nan

### ISSUE THREAD

**Issue by czz66615** [doriscullen/CodeGPTSensor#2]
04/02/2026 10:53:01 - INFO - __main__ -   epoch 0 step 100 loss nan
04/02/2026 10:53:23 - INFO - __main__ -   epoch 0 step 200 loss nan
04/02/2026 10:53:44 - INFO - __main__ -   epoch 0 step 300 loss nan
04/02/2026 10:54:06 - INFO - __main__ -   epoch 0 step 400 loss nan
04/02/2026 10:54:28 - INFO - __main__ -   epoch 0 step 500 loss nan
04/02/2026 10:54:50 - INFO - __main__ -   epoch 0 step 600 loss nan
04/02/2026 10:55:12 - INFO - __main__ -   epoch 0 step 700 loss nan
04/02/2026 10:55:34 - INFO - __main__ -   epoch 0 step 800 loss nan
04/02/2026 10:55:56 - INFO - __main__ -   epoch 0 step 900 loss nan
04/02/2026 10:56:19 - INFO - __main__ -   epoch 0 step 1000 loss nan
04/02/2026 10:56:41 - INFO - __main__ -   epoch 0 step 1100 loss nan
请问有没有环境配置文件

**Comment by doriscullen** [comment]
数据集和训练脚本已在仓库中提供，请检查是否修改了learning_rate、损失函数、数据集等。

**Comment by czz66615** [comment]
感谢解答



---原始邮件---
发件人: "Xiaodan ***@***.***&gt;
发送时间: 2026年4月2日(周四) 中午11:40
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [doriscullen/CodeGPTSensor] 为什么loss会nan (Issue #2)


doriscullen left a comment (doriscullen/CodeGPTSensor#2)
 
数据集和训练脚本已在仓库中提供，请检查是否修改了learning_rate、损失函数、数据集等。
